.travis.yml

```diff
@@ -18,7 +18,9 @@ script:
   - yarn check-format
   - yarn test
   - yarn build
+  - yarn global add codecov
 after_success:
+  - yarn codecov -f coverage/cobertura-coverage.xml
   - npm run travis-deploy-once "npm run semantic-release"
```

Comment:
why we call here npm instead of yarn?

Reply:
Just because of the history of the project, not a big deal either way but I will update it